lms/static/js/groups/views/cohorts.js

```diff
@@ -304,7 +304,6 @@
                                 });
                             }
                         }).render();
-                        this.$('#file-upload-form-file').focus();
```

Comment:
@clrux I never understood why you removed this line of code. Sorry if you explained it previously, but can you again?